code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -1408,10 +1409,19 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	// They stay locked down if their wire is cut.
 	if(wires.is_cut(WIRE_BORG_LOCKED))
 		state = TRUE
+		// Kill any in-progress override timers because a snipped wire relocks immeditely anyway.
+		deltimer(lockdown_timer)
 	if(state)
 		throw_alert("locked", /atom/movable/screen/alert/locked)
+		// Wire must be intact to override the lockdown.
+		if(!wires.is_cut(WIRE_BORG_LOCKED))
+			lockdown_timer = addtimer(CALLBACK(src,PROC_REF(lockdown_override), FALSE), 10 MINUTES, TIMER_UNIQUE | TIMER_OVERRIDE | TIMER_DELETE_ME | TIMER_STOPPABLE)
```

Comment:
Are we passing the argument `FALSE` for a reason here or is it just an accident?

Reply:
TG (from where I ported this) has this arrangement and I assume they did so for a reason.

Reply:
But the proc doesn't even have arguments?

Reply:
Removed, tested, is fine.